mev-programs/programs/vote-state/src/lib.rs

```diff
@@ -54,6 +54,7 @@ pub struct BlockTimestamp {
 #[derive(Deserialize)]
 pub enum VoteStateVersions {
     V0_23_5(Box<VoteState0_23_5>),
+    V1_14_11(Box<VoteState1_14_11>),
```

Comment:
need to double check how this works on a live network since things are shifted down and its not backwards compatible. if not backwards compatible, that would be extremely whack